test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -172,7 +173,7 @@ func (s *snapshottableTestSuite) DefineTests(driver storageframework.TestDriver,
 		})
 
 		ginkgo.Context("", func() {
-			ginkgo.It("should check snapshot fields, check restore correctly works, check deletion (ephemeral)", func() {
+			ginkgo.It("should check snapshot fields, check restore works correctly, check deletion (ephemeral)", func() {
```

Comment:
Even though this looks like a simple wording change I believe that we have some internal tests that will break if make this change, I think this change can be done later.

Reply:
Yes, these wording changes are not important for now.

Reply:
please undo this line in both tests